uint64 nrBallMeasurements
uint64[] ballObjectID
float32[] ballAzimuth
float32[] ballElevation
float32[] ballRadius
float32[] ballConfidence
float64[] ballTimestamp
rosMsgs/s_camera_type[] ballCameraType
float32[] ballCameraX
float32[] ballCameraY
float32[] ballCameraZ
float32[] ballCameraPhi

rosMsgs/BallPossession possession

bool ownRobotIsActive

uint64 nrObstacleMeasurements
uint64[] obstacleObjectID
float32[] obstacleAzimuth
float32[] obstacleElevation
float32[] obstacleRadius
float32[] obstacleConfidence
float64[] obstacleTimestamp
rosMsgs/s_camera_type[] obstacleCameraType
float32[] obstacleCameraX
float32[] obstacleCameraY
float32[] obstacleCameraZ
float32[] obstacleCameraPhi

float32 locationConfidence
float32 locationX
float32 locationY
float32 locationTheta
float32 locationVx
float32 locationVy
float32 locationVtheta
float64 locationTimestamp